bot/src/main/java/com/dongtronic/diabot/platforms/discord/commands/nightscout/NightscoutApplicationCommand.kt

```diff
@@ -186,23 +187,39 @@ class NightscoutApplicationCommand : ApplicationCommand {
     }
 
     private fun getUrl(event: SlashCommandInteractionEvent) {
-        NightscoutFacade.getUser(event.user).subscribe {
+        val missingDataMessage = "You have not configured a Nightscout URL. Use `/nightscout set url` to configure it."
+        val errorMessage = "There was an error while getting your Nightscout URL. Please try again later."
+        NightscoutFacade.getUser(event.user).subscribe({
             if (it.url != null) {
                 event.reply("Your configured Nightscout URL is `${it.url}`").setEphemeral(true).queue()
             } else {
-                event.reply("You do not have a configured Nightscout URL. Use `/nightscout set url` to configure it.").setEphemeral(true).queue()
+                event.reply(missingDataMessage).setEphemeral(true).queue()
             }
-        }
+        }, {
+            if (it is NoSuchElementException) {
+                event.reply(missingDataMessage).setEphemeral(true).queue()
+            } else {
+                replyError(event, it, errorMessage)
```

Comment:
Why event.reply() in one case, and replyError() in the other?

Reply:
replyError indicates there's a bug, it gives you a link to report it on github  
event.reply:
![event.reply](https://user-images.githubusercontent.com/11539403/198948625-f3f4de1b-f0d2-4252-b11e-999b531e29c8.png)  

replyError:
![replyError](https://user-images.githubusercontent.com/11539403/198948747-41e7069d-28c8-4a8d-9031-7a8dc28d8a60.png)